src/modules/menu/components/menu/feedback/FeedbackButton.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+    <HeaderLink
+        v-if="showAsLink"
+        :primary="true"
+        data-cy="feedback-link-button"
+        @click="showFeedbackForm = true"
+    >
+        <strong>{{ $t('test_map_give_feedback') }}</strong>
+    </HeaderLink>
+    <button
+        v-else
+        class="btn btn-primary btn-sm mx-1"
+        data-cy="feedback-button"
+        @click="showFeedbackForm = true"
+    >
+        {{ $t('test_map_give_feedback') }}
+    </button>
+    <ModalWithBackdrop
+        v-if="showFeedbackForm"
+        :title="$t('feedback_rating_title')"
+        @close="showFeedbackForm = false"
+    >
+        <div class="p-2" data-cy="feedback-form">
+            <FeedbackRating
+                class="mb-3 text-center"
+                :max-rating="maxRating"
+                @rating-change="ratingChange"
+            />
+            <textarea
+                v-model="feedback"
+                class="form-control"
+                data-cy="feedback-text"
+                :placeholder="$t('feedback_rating_text')"
+            ></textarea>
+            <div class="my-4">
+                <!-- eslint-disable vue/no-v-html-->
+                <small v-html="$t('feedback_disclaimer')" />
+                <!-- eslint-enable vue/no-v-html-->
+            </div>
+            <div class="text-end">
+                <button class="btn btn-light mx-2" @click="closeAndCleanForm">
+                    <span v-if="request.completed">{{ $t('close') }}</span>
+                    <span v-else>{{ $t('cancel') }}</span>
+                </button>
+                <button
+                    :disabled="!feedbackCanBeSent"
+                    class="btn btn-primary"
+                    data-cy="submit-feedback-button"
+                    @click="sendFeedback"
+                >
+                    <FontAwesomeIcon
+                        v-if="request.pending"
+                        icon="spinner"
+                        pulse
+                        data-cy="feedback-pending-icon"
+                    />
+                    <FontAwesomeIcon
+                        v-else-if="request.completed"
+                        icon="check"
+                        data-cy="feedback-success-icon"
+                    />
+                    <span v-else data-cy="feedback-send-text">{{ $t('send') }}</span>
+                </button>
+            </div>
+            <div v-if="request.completed || request.failed" class="text-end mt-3">
+                <span>
+                    <small v-if="request.failed" class="text-danger" data-cy="feedback-failed-text">
+                        {{ $t('upload_failed') }}
```

Comment:
I would create a new translation for this:

`send_failed`

| Lang | Translation | 
| en | Sending failed |
| fr | Envoi échoué |